### ISSUE TITLE
List of classes/files/functions that may no longer be used/tested and can be marked for removal

### ISSUE THREAD

**Issue by bartgol** [SNLComputation/Albany#448]
As the title says, this issue is meant to track files/classes/functions/etc. that are no longer used and tested. They become a burden to maintain, and may not even be fully working or working correctly. As such, it is better for Albany to purge them.

Please, edit this post to add new items, or ot cross off items that have been removed. If an item has to be kept, do not cross it off (or else someone may add it back later); instead, comment next to it, so we know why we should keep it.

- SaveEigenData and the likes.

**Comment by ikalash** [comment]
I would say SaveEigenData can be removed for sure.  The only application that was using it to my knowledge is QCAD, and that is no longer a part of the master branch.

**Comment by bartgol** [comment]
What about the rest? I found a problem in ATO (`ATO/problems/LinearElasticityModalProblem.hpp`) that uses the GatherEigenData evaluator, so I guess we may have to keep it. Although, I don't know if that problem in ATO is 'really' used...

**Comment by ikalash** [comment]
Oh shoot...  I missed that one somehow.  This is a question for @jrobbin .  Josh, based on our email exchange a few weeks ago, ATO is still under development, but I'm not sure to what extent, and whether you need all the capabilities that are currently there.  Are you still interested in the LinearElasticityModalProblem?  We're looking to refactor/clean up Albany, which includes removing obsolete code.  I had though the EigenData stuff was obsolete as QCAD is no longer in master, but it appears you are using it for the LinearElasticityModalProblem.

**Comment by bartgol** [comment]
I stumbled on this issue. Now that pruning of non-LandIce related stuff is happening, this might be relevant.

In particular, I would like to ask you guys what's the plan with Rythmos. Are we ok supporting just tempus? I assume the answer is yes, but I wanted to double check.

**Comment by mperego** [comment]
I think the plan is to support only Tempus, but we first need to convert all the tests using Rythmos to Tempus. Probably we don't have to keep all the tests using Rythmos, but some might be useful. Anyway, it's not urgent, I think.

**Comment by ikalash** [comment]
Right, there's no need to keep using Rythmos.  I agree with what Mauro said, namely that we should switch the existing Rythmos tests to Tempus.  I could do this.  How many Rythmos tests are there in the current master?

**Comment by bartgol** [comment]
I haven't counted them. I'm guessing about 10 or so, mostly in the basic and demo tests. But it's ok, we can wait.

**Comment by ikalash** [comment]
Now that the VPN is back up, I am able to check myself.  It's more like 15.  I can go ahead and convert these to Tempus sometime in the near future.  I'll open a separate issue.
